requirements.txt

```diff
@@ -20,9 +20,7 @@ incremental==17.5.0
 markdown==2.6.11
 msgpack==0.5.6
 psycopg2==2.7.4
-pypiwin32==223

```

Comment:
Removed because of problem with this on linux, so you installing it through bat file and it is not needed here